LICENSE.md

```diff
@@ -1,16 +1,31 @@
+BSD 3-Clause License
```

Comment:
Use either short identifier (BSD-3-Clause) or Full name (BSD 3-Clause "New" or "Revised" License)

Reply:
I recommend the short identifier.

Reply:
Done

Reply:
This